### PR TITLE
Adding some conferences from papercall

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,7 +238,9 @@ All the data (past and coming) are available publicly in JSON:
 * 25: [KCD Romania 2024](https://kcdromania.ro/) - Bucharest (Romania) <a href="https://sessionize.com/kcd-romania-2024"><img alt="CFP KCD Romania 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-Jan-2024&color=red"></a>
 * 25-26: [MiXiT](https://mixitconf.org/fr/) - Lyon (France) <a href="https://mixitconf.org/fr/"><img alt="CFP MiXiT" src="https://img.shields.io/static/v1?label=CFP&message=from%2016-Dec-2023-until%2019-Jan-2024&color=red"></a> <img alt="Closed Captions" src="https://img.shields.io/static/v1?label=CC&message=Closed%20Captions&color=blue" />
 * 25-26: [Android Makers](https://androidmakers.droidcon.com/) - Paris (France) <a href="https://sessionize.com/androidmakers-by-droidcon-2024"><img alt="CFP Android Makers by droidcon 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2001-March-2024&color=green"></a>
+* 25-26: [Vue.js Live Conference](https://vuejslive.com/) - Online <a href="https://www.papercall.io/vuejs-conference"><img alt="CFP Vue.js Live Conference " src="https://img.shields.io/static/v1?label=CFP&message=until%2015-March-2024&color=green"></a>
```
Comment (scraly) [inline README.md:241]:
```suggestion
* 25-26: [Vue.js Live Conference](https://vuejslive.com/) - Online <a href="https://www.papercall.io/vuejs-conference"><img alt="CFP Vue.js Live Conference" src="https://img.shields.io/static/v1?label=CFP&message=until%2015-March-2024&color=green"></a>
```

File: README.md [hunk 2]
```diff
@@ -238,7 +238,9 @@ All the data (past and coming) are available publicly in JSON:
...
 * 25-27: [BSides Goa Security Conference](https://www.bsidesgoa.in/) - Goa (India) <a href="https://www.bsidesgoa.in/submit-cfp"><img alt="CFP BSides Goa Security Conference" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-January-2024&color=red"></a>
+* 26-27: [BSides Colombia 2024](https://bsidesco.org/) - Medellin (Colombia) <a href="https://www.papercall.io/bsidesco2024"><img alt="CFP BSides Colombia 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2020-March-2024&color=green"></a>
```
Comment (scraly) [inline README.md:243]:
```suggestion
* 26-27: [BSides Colombia 2024](https://bsidesco.org/) - Medellin (Colombia) <a href="https://www.papercall.io/bsidesco2024"><img alt="CFP BSides Colombia 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2020-March-2024&color=green"></a>
```

File: README.md [hunk 3]
```diff
@@ -277,6 +280,7 @@ All the data (past and coming) are available publicly in JSON:
 * 13-16: [JCON EUROPE 2024](https://2024.europe.jcon.one/) - Cologne (Germany) <a href="https://sessionize.com/jcon-europe-2024"><img alt="CFP JCON EUROPE 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-Nov-2023&color=red"></a>
 * 14-15: [DevOpsDays Seattle](https://devopsdays.org/events/2024-seattle) - Seattle (USA) <a href="https://talks.devopsdays.org/devopsdays-seattle-2024/cfp"><img alt="CFP DevOpsDays Seattle 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2015-Jan-2023&color=red"></a>
 * 14-15: [DevOpsDays Philadelphia](https://devopsdays.org/events/2024-philadelphia) - Philadelphia (USA)
+* 14-16: [OpenSIPS Summit 2024](https://www.opensips.org/events/Summit-2024Valencia) - Valencia (Spain) <a href="https://www.papercall.io/opensips-summit-2024"><img alt="CFP OpenSIPS Summit 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2008-March-2024&color=green"></a>
```
Comment (scraly) [inline README.md:283]:
```suggestion
* 14-16: [OpenSIPS Summit 2024](https://www.opensips.org/events/Summit-2024Valencia) - Valencia (Spain) <a href="https://www.papercall.io/opensips-summit-2024"><img alt="CFP OpenSIPS Summit 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2008-March-2024&color=green"></a>
```

File: README.md [hunk 4]
```diff
@@ -289,10 +293,12 @@ All the data (past and coming) are available publicly in JSON:
 * 16-17: [DevSum](https://www.devsum.se/) - Stockholm (Sweden) <a href="https://sessionize.com/devsum-2024"><img alt="CFP DevSum" src="https://img.shields.io/static/v1?label=CFP&message=until%2003-Dec-2023&color=red"></a>
 * 16-17: [PHP Day](https://2024.phpday.it/) - Verona (Italy) <a href="https://sessionize.com/phpday-2024"><img alt="CFP PHP Day" src="https://img.shields.io/static/v1?label=CFP&message=until%2014-Jan-2024&color=red"></a>
 * 16-17: [DefCamp Cluj-Napoca](https://cluj.def.camp) - Cluj-Napoca (Romania) <a href="https://sessionize.com/defcampcluj"><img alt="CFP DefCamp Cluj-Napoca" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-March-2024&color=green"></a>
+* 17-18: [JavaDay Lviv 2024](http://www.javaday.org.ua) - Lviv (Ukraine) <a href="https://www.papercall.io/java-day-lviv-2024"><img alt="CFP JavaDay Lviv 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2027-March-2024&color=green"></a>
```
Comment (scraly) [inline README.md:296]:
```suggestion
* 17-18: [JavaDay Lviv 2024](http://www.javaday.org.ua) - Lviv (Ukraine) <a href="https://www.papercall.io/java-day-lviv-2024"><img alt="CFP JavaDay Lviv 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2027-March-2024&color=green"></a>
```

File: README.md [hunk 5]
```diff
@@ -377,11 +383,13 @@ All the data (past and coming) are available publicly in JSON:
...
 * 14: [UAD360](https://uad360.es/) - Malaga (Spain) <a href="https://uad360.es/ponentes"><img alt="CFP UAD360" src="https://img.shields.io/static/v1?label=CFP&message=until%2001-April-2024&color=green"></a>
 * 14: [React Norway](https://reactnorway.com/) - Larvik (Norway) <a href="https://docs.google.com/forms/d/e/1FAIpQLSc_xjLBFkK0N1i_r7MgOpw4JMKT7SyapJBRG1jQaiBRfxHTgg/viewform"><img alt="CFP React Norway" src="https://img.shields.io/static/v1?label=CFP&message=until%2001-April-2024&color=green"></a>
 * 14-18: [React Summit](https://reactsummit.com/) - Amsterdam (Netherlands) <a href="https://docs.google.com/forms/d/e/1FAIpQLSfeqfjOglU2yyJtVK7AR5gt_tkby8YVXNCCywIMSc9npGlnLA/viewform"><img alt="CFP React Summit" src="https://img.shields.io/static/v1?label=CFP&message=until%2003-March-2024&color=green"></a>
+* 15-19: [TechLead Conference](https://techleadconf.com/) - Online <a href="https://www.papercall.io/techlead"><img alt="CFP TechLead Conference " src="https://img.shields.io/static/v1?label=CFP&message=until%2031-March-2024&color=green"></a>
```
Comment (scraly) [inline README.md:392]:
```suggestion
* 15-19: [TechLead Conference](https://techleadconf.com/) - Online <a href="https://www.papercall.io/techlead"><img alt="CFP TechLead Conference" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-March-2024&color=green"></a>
```

File: README.md [hunk 6]
```diff
@@ -404,9 +412,11 @@ All the data (past and coming) are available publicly in JSON:
 * 20-21: [Golang Conf 2024](https://cfp.golangconf.ru/en) - St. Petersburg (Rusia) <a href="https://cfp.golangconf.ru/en"><img alt="CFP Golang Conf 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2015-February-2024&color=green"></a>
 * 21-22: [Code Beam Lite](https://www.codebeamcorunha.es/en) - Coruna (Spain) <a href="https://sessionize.com/code-beam-corunha-2024/"><img alt="CFP Code Beam Lite 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-January-2024&color=green"></a>
 * 21-22: [Opensouthcode](https://www.opensouthcode.org) - Malaga (Spain) <a href="https://www.opensouthcode.org/conferences/opensouthcode2024/program/proposals/new"><img alt="CFP Opensouthcode 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-March-2024&color=green"></a>
+* 21-22: [Play Secure 2024](https://playsecure.online) - Online <a href="https://www.papercall.io/playsecure2024"><img alt="CFP Play Secure 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2015-March-2024&color=green"></a>
```
Comment (scraly) [inline README.md:415]:
```suggestion
* 21-22: [Play Secure 2024](https://playsecure.online) - Online <a href="https://www.papercall.io/playsecure2024"><img alt="CFP Play Secure 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2015-March-2024&color=green"></a>
```

File: README.md [hunk 7]
```diff
@@ -404,9 +412,11 @@ All the data (past and coming) are available publicly in JSON:
...
 * 24-25: [InfoQ Dev Summit Boston](https://devsummit.infoq.com/boston_medialisting2024) - Boston, MA (USA)
 * 24-25: [WasmCon](https://events.linuxfoundation.org/wasmcon/) - Seattle (USA)
 * 24-27: [psconf.eu 2024](https://psconf.eu) - Antwerp (Belgium) <a href="https://sessionize.com/psconfeu-2024/"><img alt="CFP psconf.eu 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2016-December-2023&color=red"></a>
+* 25-27: [The Perl & Raku Conference](https://tprc.us/) - Las Vegas, NV (USA) <a href="https://www.papercall.io/tprclas24"><img alt="CFP The Perl & Raku Conference in Las Vegas, NV 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2006-April-2024&color=green"></a>
```
Comment (scraly) [inline README.md:419]:
```suggestion
* 25-27: [The Perl & Raku Conference](https://tprc.us/) - Las Vegas, NV (USA) <a href="https://www.papercall.io/tprclas24"><img alt="CFP The Perl & Raku Conference in Las Vegas, NV 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2006-April-2024&color=green"></a>
```

File: README.md [hunk 8]
```diff
@@ -428,13 +438,16 @@ All the data (past and coming) are available publicly in JSON:
...
 * 8-10: [Riviera DEV](https://rivieradev.fr/) - Sophia Antipolis (France) <a href="https://conference-hall.io/public/event/ieJBZ8FhXHMinZmeTCJp"><img alt="CFP Riviera Dev 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2005-March-2024&color=green"></a>
 * 8-11: [useR!](https://events.linuxfoundation.org/user/) - Salzburg (Austria) <a href="https://sessionize.com/user-2024/"><img alt="CFP useR!" src="https://img.shields.io/static/v1?label=CFP&message=until%2010-March-2024&color=green"></a>
+* 9-10: [DevOpsDays Nashville 2024](https://devopsdays.org/events/2024-nashville/welcome/) - Nashville, TN (USA) <a href="https://www.papercall.io/devopsdays-nashville-2024"><img alt="CFP DevOpsDays Nashville 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2031-May-2024&color=green"></a>
```
Comment (scraly) [inline README.md:441]:
```suggestion
* 9-10: [DevOpsDays Nashville 2024](https://devopsdays.org/events/2024-nashville/welcome/) - Nashville, TN (USA) <a href="https://www.papercall.io/devopsdays-nashville-2024"><img alt="CFP DevOpsDays Nashville 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-May-2024&color=green"></a>
```

File: README.md [hunk 9]
```diff
@@ -428,13 +438,16 @@ All the data (past and coming) are available publicly in JSON:
...
 * 17-19: [Chain React 2024](http://ChainReactConf.com) - Portland, OR (USA) <a href="https://sessionize.com/chain-react-2024/"><img alt="CFP Chain React 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2003-May-2024&color=green"></a>
 * 17-19: [WeAreDevelopers World Congress 2024](https://worldcongress.dev) - Berlin (Germany) <a href="https://sessionize.com/wearedevelopers-world-congress-2024"><img alt="CFP WeAreDevelopers World Congress 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-January-2024&color=red"></a>
 * 18-19: [DevRel Con New York](https://nyc24.devrelcon.dev/) - New York (USA) <a href="https://nyc24.devrelcon.dev/call-for-proposals/"><img alt="CFP DevRel Con NY 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2001-March-2024&color=green"></a>
+* 18-19: [AI in Production](https://www.aiinproduction.com) - Asheville, NC (USA) <a href="https://www.papercall.io/aiinproduction"><img alt="CFP AI in Production " src="https://img.shields.io/static/v1?label=CFP&message=until%2030-April-2024&color=green"></a>
```
Comment (scraly) [inline README.md:450]:
```suggestion
* 18-19: [AI in Production](https://www.aiinproduction.com) - Asheville, NC (USA) <a href="https://www.papercall.io/aiinproduction"><img alt="CFP AI in Production" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-April-2024&color=green"></a>
```

File: README.md [hunk 10]
```diff
@@ -443,6 +456,7 @@ All the data (past and coming) are available publicly in JSON:
 * 25-27: [The Bitcoin Conference 2024](https://b.tc/conference) - Nashville, Tennessee (USA)
 * 26-28: [CommunityOverCode Asia 2024](https://asia.communityovercode.org/) - Hangzhou (China) <a href="https://sessionize.com/communityovercode-asia-2024"><img alt="CFP CommunityOverCode Asia 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2021-April-2024&color=green"></a>
 * 27: [How.Camp 2024: A day of FOSS](https://how.camp) - Gabrovo (Bulgaria)
+* 27: [Bsides Exeter 2024](https://bsidesexeter.co.uk) - Exeter (UK) <a href="https://www.papercall.io/bsidesexeter2024"><img alt="CFP Bsides Exeter 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2024-March-2024&color=green"></a>
```
Comment (scraly) [inline README.md:459]:
```suggestion
* 27: [Bsides Exeter 2024](https://bsidesexeter.co.uk) - Exeter (UK) <a href="https://www.papercall.io/bsidesexeter2024"><img alt="CFP Bsides Exeter 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2024-March-2024&color=green"></a>
```

File: README.md [hunk 11]
```diff
@@ -463,17 +477,21 @@ All the data (past and coming) are available publicly in JSON:
 
 * 3-4: [ContainerDays Conference 2024](https://www.containerdays.io/containerdays-conference-2024/) - Hamburg (Germany) <a href="https://sessionize.com/CDS-conference-2024/"><img alt="CFP ContainerDays Conference 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-March-2024&color=green"></a>
 * 4-5: [JavaZone](https://2024.javazone.no/) - Oslo (Norway) <a href="https://2024.javazone.no/speakers"><img alt="CFP JavaZone 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2008-April-2024&color=green"></a>
+* 4-6: [BITCON 2024](https://bitcon.blacksintechnology.net) - Atlanta, GA (USA) <a href="https://www.papercall.io/bitcon-2024"><img alt="CFP BITCON 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2031-May-2024&color=green"></a>
```
Comment (scraly) [inline README.md:480]:
```suggestion
* 4-6: [BITCON 2024](https://bitcon.blacksintechnology.net) - Atlanta, GA (USA) <a href="https://www.papercall.io/bitcon-2024"><img alt="CFP BITCON 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-May-2024&color=green"></a>
```

File: README.md [hunk 12]
```diff
@@ -463,17 +477,21 @@ All the data (past and coming) are available publicly in JSON:
...
 * 5-6: [React Universe Conf 2024](http://reactuniverseconf.com/) - Wrocław (Poland) <a href="https://sessionize.com/reactuniverseconf2024"><img alt="CFP React Universe Conf 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2022-March-2024&color=green"></a>
+* 5-6: [OWASP New Zealand Day](https://appsec.org.nz/conference) - Auckland (New Zealand) <a href="https://www.papercall.io/owaspnz2024"><img alt="CFP OWASP New Zealand Day " src="https://img.shields.io/static/v1?label=CFP&message=until%2030-June-2024&color=green"></a>
```
Comment (scraly) [inline README.md:482]:
```suggestion
* 5-6: [OWASP New Zealand Day](https://appsec.org.nz/conference) - Auckland (New Zealand) <a href="https://www.papercall.io/owaspnz2024"><img alt="CFP OWASP New Zealand Day" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-June-2024&color=green"></a>
```

File: README.md [hunk 13]
```diff
@@ -463,17 +477,21 @@ All the data (past and coming) are available publicly in JSON:
...
 * 6: [JUG Summer Camp](https://www.jugsummercamp.org) - La Rochelle (France)
 * 6-8: [Blue Team Con](https://blueteamcon.com/) - Chicago (USA) <a href="https://blueteamcon.com/blue-team-con-2024-call-for-papers/"><img alt="CFP Blue Team Con" src="https://img.shields.io/static/v1?label=CFP&message=until%2017-March-2024&color=green"></a>
 * 9-12: [NDC TechTown 2024](https://ndctechtown.com) - Kongsberg (Norway) <a href="https://sessionize.com/ndc-techtown-2024"><img alt="CFP NDC TechTown 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2014-April-2024&color=green"></a>
+* 11-13: [EuRuKo 2024](https://2024.euruko.org) - Sarajevo (Bosnia and Herzegovina) <a href="https://www.papercall.io/euruko2024"><img alt="CFP EuRuKo 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2015-April-2024&color=green"></a>
```
Comment (scraly) [inline README.md:486]:
```suggestion
* 11-13: [EuRuKo 2024](https://2024.euruko.org) - Sarajevo (Bosnia and Herzegovina) <a href="https://www.papercall.io/euruko2024"><img alt="CFP EuRuKo 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2015-April-2024&color=green"></a>
```

File: README.md [hunk 14]
```diff
@@ -463,17 +477,21 @@ All the data (past and coming) are available publicly in JSON:
...
 * 16-17: [Linux Security Summit Europe](https://events.linuxfoundation.org/linux-security-summit-europe/) - Vienna (Austria)
 * 16-18: [Open Source Summit Europe](https://events.linuxfoundation.org/open-source-summit-europe/) - Vienna (Austria)
 * 16-19: [Workplace Ninja Summit 2024](https://wpninjas.eu) - Luzern (Switzerland) <a href="https://sessionize.com/workplace-ninja-summit-2024"><img alt="CFP Workplace Ninja Summit 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2014-April-2024&color=green"></a>
 * 18: [You Got This Fall 2024](https://yougotthis.io/events/fall-2024) - Online
 * 18-19: [API Days London](https://www.apidays.global/london/) - London (UK) <a href="https://apidays.typeform.com/to/ILJeAaV8"><img alt="CFP API Days London" src="https://img.shields.io/static/v1?label=CFP&message=until%2018-August-2024&color=green"></a>
+* 18-20: [Replay 2024](https://temporal.io/replay) - Bellevue, WA (USA) <a href="https://www.papercall.io/replay2024"><img alt="CFP Replay 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2031-May-2024&color=green"></a>
```
Comment (scraly) [inline README.md:494]:
```suggestion
* 18-20: [Replay 2024](https://temporal.io/replay) - Bellevue, WA (USA) <a href="https://www.papercall.io/replay2024"><img alt="CFP Replay 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-May-2024&color=green"></a>
```

File: README.md [hunk 15]
```diff
@@ -486,9 +504,11 @@ All the data (past and coming) are available publicly in JSON:
 * 25-27: [UX Healthcare Europe](https://uxhealthcare.co/) - Prague (Czech Republic)
 * 25-26: [JSConf Ireland 2024](https://jsconf.ie/) - Dublin (Ireland) <a href="https://www.jsconf.ie/call-for-proposals/"><img alt="CFP JSConf Ireland 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2029-February-2024&color=green"></a>
 * 26: [Conf42 Kube Native](https://www.conf42.com/kubenative2024) - Online <a href="https://www.papercall.io/conf42-kube-native-2024"><img alt="CFP Conf42 Kube Native" src="https://img.shields.io/static/v1?label=CFP&message=until%2026-Aug-2024&color=green"></a>
+* 26: [AWS Community Summit Manchester 2024](http://www.comsum.co.uk) - Manchester (UK) <a href="https://www.papercall.io/comsum24"><img alt="CFP AWS Community Summit Manchester 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2031-July-2024&color=green"></a>
```
Comment (scraly) [inline README.md:507]:
```suggestion
* 26: [AWS Community Summit Manchester 2024](http://www.comsum.co.uk) - Manchester (UK) <a href="https://www.papercall.io/comsum24"><img alt="CFP AWS Community Summit Manchester 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-July-2024&color=green"></a>
```

File: README.md [hunk 16]
```diff
@@ -486,9 +504,11 @@ All the data (past and coming) are available publicly in JSON:
...
 * 26-27: [DevOpsDays London](https://devopsdays.org/events/2024-london/welcome/) - London (UK) <a href="https://docs.google.com/forms/d/e/1FAIpQLSdUK4VNyhpfpqhrm8BY7KmAvSNQHIuL49YOabkYDInbnyKD1g/viewform"><img alt="CFP DevOpsDays London" src="https://img.shields.io/static/v1?label=CFP&message=until%2024-May-2024&color=green"></a>
 * 26-27: [Rails World 2024](https://rubyonrails.org/world) - Toronto (Canada)
 * 26-27: [Less Conference Madrid](https://less.works/conferenza/conferences/2024-less-conference-madrid-10) - Madrid (Spain) <a href="https://less.works/conferenza/conferences/2024-less-conference-madrid-10/submission"><img alt="CFP Less Conference Madrid" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-March-2024&color=green"></a>
+* 26-27: [ServerSide.swift 2024 ](https://www.serversideswift.info) - London (UK) <a href="https://www.papercall.io/server-side-swift-2024"><img alt="CFP ServerSide.swift 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2031-May-2024&color=green"></a>
```
Comment (scraly) [inline README.md:511]:
```suggestion
* 26-27: [ServerSide.swift 2024 ](https://www.serversideswift.info) - London (UK) <a href="https://www.papercall.io/server-side-swift-2024"><img alt="CFP ServerSide.swift 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-May-2024&color=green"></a>
```

File: README.md [hunk 17]
```diff
@@ -502,6 +522,7 @@ All the data (past and coming) are available publicly in JSON:
 * 10-11: [Volcamp](https://www.volcamp.io) - Clermont-Ferrand (France)
 * 10-11: [Forum PHP](https://event.afup.org/) - Marne-la-Vallée (France)
 * 10-11: [Zebu Live 2024](https://zebulive.xyz) - London (UK) <a href="https://sessionize.com/zebu-live-2024"><img alt="CFP Zebu Live 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-May-2024&color=green"></a>
+* 10-11: [EuroRust 2024](https://eurorust.eu) - Vienna (Austria) <a href="https://www.papercall.io/eurorust-2024"><img alt="CFP EuroRust 2024 – Vienna, Austria " src="https://img.shields.io/static/v1?label=CFP&message=until%2003-June-2024&color=green"></a>
```
Comment (scraly) [inline README.md:525]:
```suggestion
* 10-11: [EuroRust 2024](https://eurorust.eu) - Vienna (Austria) <a href="https://www.papercall.io/eurorust-2024"><img alt="CFP EuroRust 2024 – Vienna, Austria" src="https://img.shields.io/static/v1?label=CFP&message=until%2003-June-2024&color=green"></a>
```

File: README.md [hunk 18]
```diff
@@ -514,6 +535,7 @@ All the data (past and coming) are available publicly in JSON:
 * 17-18: [DevFest Nantes](https://devfest.gdgnantes.com/en/) - Nantes (France)
 * 17-18: [DotAI](https://www.dotai.io/) - Paris (France) <a href="https://www.dotai.io/cfp"><img alt="CFP DotAI 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-April-2024&color=green"></a>
 * 17-18: [Rubyfuza](https://dev.events/conferences/rubyfuza-cape-town-9-2024) - Cape Town (South Africa)
+* 17-18: [Rubyfuza & friends 2024](https://www.rubyfuza.org/) - Cape Town (South Africa) <a href="https://www.papercall.io/rubyfuza-2024"><img alt="CFP Rubyfuza & friends 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2010-August-2024&color=green"></a>
```
Comment (scraly) [inline README.md:538]:
```suggestion
* 17-18: [Rubyfuza & friends 2024](https://www.rubyfuza.org/) - Cape Town (South Africa) <a href="https://www.papercall.io/rubyfuza-2024"><img alt="CFP Rubyfuza & friends 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2010-August-2024&color=green"></a>
```

File: README.md [hunk 19]
```diff
@@ -542,12 +564,14 @@ All the data (past and coming) are available publicly in JSON:
 * 21: [Conf42 Prompt Engineering 2024](https://www.conf42.com/prompt2024) - Online <a href="https://www.papercall.io/conf42-prompt-engineering-2024"><img alt="CFP Conf42 Prompt Engineering 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2021-October-2024&color=green"></a>
 * 22-23: [Open Conf - 2024](https://www.open-conf.gr/) - Athens (Greece) <a href="https://sessionize.com/open-conf-2024"><img alt="CFP Open Conf - 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2029-March-2024&color=green"></a>
 * 28: [ESRM UK](https://whitehallmedia.co.uk/future-events/) - London (UK)
+* 28: [SREday 2024](https://www.sreday.com) - Amsterdam (Netherlands) <a href="https://www.papercall.io/sreday-2024-amsterdam"><img alt="CFP SREday 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2029-August-2024&color=green"></a>
```
Comment (scraly) [inline README.md:567]:
```suggestion
* 28: [SREday 2024](https://www.sreday.com) - Amsterdam (Netherlands) <a href="https://www.papercall.io/sreday-2024-amsterdam"><img alt="CFP SREday 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2029-August-2024&color=green"></a>
```

File: README.md [hunk 20]
```diff
@@ -542,12 +564,14 @@ All the data (past and coming) are available publicly in JSON:
...
 
 ### December
 
 * 03-04: [Frontend Barcelona 2024](https://frontend.barcelona/?utm_source=developers-conference-agenda&utm_medium=link&utm_campaign=frontend-barcelona-2024&utm_id=frontend-barcelona-2024) - Barcelona (Spain) <a href="https://frontend.barcelona/call-for-papers"><img alt="CFP Frontend Barcelona" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-September-2024&color=green"></a>
 * 11: [You Got This Winter 2024](https://yougotthis.io/events/winter-2024) - Online
 * 11-12: [KubeCon + CloudNativeCon India 2024](https://events.linuxfoundation.org/kubecon-cloudnativecon-india/) - Delhi (India) 
+* 14: [DevOpsDays Recife 2024](https://devopsdays.org/recife) - Recife (Brazil) <a href="https://www.papercall.io/dodrec24"><img alt="CFP DevOpsDays Recife 2024 " src="https://img.shields.io/static/v1?label=CFP&message=until%2001-July-2024&color=green"></a>
```
Comment (scraly) [inline README.md:574]:
```suggestion
* 14: [DevOpsDays Recife 2024](https://devopsdays.org/recife) - Recife (Brazil) <a href="https://www.papercall.io/dodrec24"><img alt="CFP DevOpsDays Recife 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2001-July-2024&color=green"></a>
```

**Review by scraly — CHANGES_REQUESTED** [review]
thanks
just need to trim CFP title :)

**Comment by jespino** [comment]
Oops, fixed. maybe I need a `strings.TrimSpace` somewhere in some code that I have laying around in my PC.

**Comment by scraly** [comment]
thx! :)

**Pull request merged**